lld/ELF/Arch/AArch64.cpp

```diff
@@ -221,6 +221,7 @@ RelType AArch64::getDynRel(RelType type) const {
 
 int64_t AArch64::getImplicitAddend(const uint8_t *buf, RelType type) const {
   switch (type) {
+    // Relocation types that point at data.
```

Comment:
I think this comment can be omitted. It might be useful to add a comment to state where code relocations start.